data/recommendation/style.scss

```diff
@@ -13,6 +13,9 @@ h1 {
 }
 
 .sync-box {
+  &.hidden {
+    display: none;
+  }
```

Comment:
It is probably worth having a top level rule for this. I think it is fair to say that `class="hidden"` should map to `display: none` in almost every case.